packages/xchain-bitcoin/src/client.ts

```diff
@@ -253,44 +239,41 @@ class Client implements BitcoinClient, XChainClient {
    * @returns {TxsPage} The transaction history.
    */
   getTransactions = async (params?: TxHistoryParams): Promise<TxsPage> => {
-    const limit = params?.limit ?? 10
+    // Sochain API doesn't have pagination parameter
     const offset = params?.offset ?? 0
+    const limit = params?.limit || 10
 
-    let totalCount = 0
-    const transactions: Tx[] = []
     try {
       const address = params?.address ?? this.getAddress()
-      //Calling getAddress without limit/offset to get total count
-      const dAddr = await blockChair.getAddress(this.nodeUrl, address, this.nodeApiKey)
-      totalCount = dAddr[address].transactions.length
 
-      const dashboardAddress = await blockChair.getAddress(this.nodeUrl, address, this.nodeApiKey, limit, offset)
-      const txList = dashboardAddress[address].transactions
+      const response = await sochain.getAddress(this.nodeUrl, this.net, address)
+      const total = response.txs.length
+      const transactions: Tx[] = []
 
-      for (const hash of txList) {
-        const rawTx = (await blockChair.getTx(this.nodeUrl, hash, this.nodeApiKey))[hash]
+      const txs = response.txs.filter((_, index) => offset <= index && index < offset + limit)
+      for (const txItem of txs) {
+        const rawTx = await sochain.getTx(this.nodeUrl, this.net, txItem.txid)
         const tx: Tx = {
           asset: AssetBTC,
-          from: rawTx.inputs.map((i: TxIO) => ({ from: i.recipient, amount: baseAmount(i.value, 8) })),
+          from: rawTx.inputs.map((i) => ({ from: i.address, amount: assetToBase(assetAmount(i.value, 8)) })),
```

Comment:
Provide  `BTC_DECIMAL`  and use it instead of `8` everywhere. 
```ts
assetAmount(i.value, BTC_DECIMAL)
```
Pls make `BTC_DECIMAL` accessible from "outside" world (similar to `XYZ_DECIMAL` in other `xchain-*` packages) .